### PR TITLE
Scripts/Command: implement .debug play music command

### PR DESCRIPTION
**Pull request by ForesterDev**
**Changes proposed:**

-  implement .debug play music command

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Tests performed:** (Does it build, tested in-game, etc.)
Tested in game, build

### CODE REVIEW

File: src/server/scripts/Commands/cs_debug.cpp [hunk 1]
```diff
@@ -202,15 +203,42 @@ class debug_commandscript : public CommandScript
...
+        // USAGE: .debug play music #musicId
+        // #musicId - ID decimal number from SoundEntries.dbc (1st column)
+        if (!*args)
+        {
+            handler->SendSysMessage(LANG_BAD_VALUE);
+            handler->SetSentErrorMessage(true);
+            return false;
+        }
+
+        uint32 musicId = atoi((char*)args);
```
Comment (Rochet2) [inline src/server/scripts/Commands/cs_debug.cpp]: could use `atoul(args);`

**Comment by Kittnz** [comment]
Nice, i didn't bother implementing this for debug because the soundsid that you can play are the same. Only difference for this opcode is that it plays on the music channel (client).

**Comment by tkrokli** [comment]
@ForesterDev : do you mind adding the SQL file name `2017_05_25_00_auth.sql` to the `INSERT INTO updates` line in your `sql/base/auth_database.sql` file in this PR branch? I think it would make editing during merge a bit easier.

**Review by Shauren — APPROVED** [review]

**Comment by tkrokli** [comment]
Cosmetic details: 
sql/updates/world/3.3.5/2017_05_25_99_world.sql :
add the missing `than` to `Warning: client may have more 5000 sounds...');`
to make it look like this : `Warning: client may have more than 5000 sounds...');`

Also: have you tested that the command shows proper line breaks in-game? Normally the line breaks are replaced with `\n` in the SQL insert. (I haven't tested your version yet.)

`sql/updates/auth/3.3.5/2017_05_25_00_auth.sql`
Redundant blank line at the end of the file (not very important, mostly codestyle).

File: sql/updates/world/3.3.5/2017_05_25_99_world.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+DELETE FROM `command` WHERE `name` = 'debug play music';
+INSERT INTO `command` (`name`,`permission`,`help`) VALUES
+('debug play music', 318, 'Syntax: .debug play music #musicId
+
+Play music with #musicId.
+Music will be play only for you. Other players do not hear this.
+Warning: client may have more 5000 sounds...');
```
Comment (Aokromes) [inline sql/updates/world/3.3.5/2017_05_25_99_world.sql]: i don't see this warning needed.

**Pull request closed without merging**